src/core/analysis/stoppingRule/GelmanRubinStoppingRule.cpp

```diff
@@ -104,7 +104,7 @@ bool GelmanRubinStoppingRule::stop( size_t g )
         num_variables_in_rep = data[i].size();
         
         // if we have different numbers of variables in different replicates, something has gone terribly wrong
-        if (i > 1 && data[i].size() != num_variables_in_rep)
+        if (i > 0 && data[i].size() != num_variables_in_rep)
         {
             throw RbException("The replicates contain different sets of parameters.");
```

Comment:
Is it overkill to also check that the parameter names are equivalent?

Reply:
Maybe not, but I'm not sure it should be done by every individual stopping rule... if anything, I'm wondering if the check is needed at all. Even when manipulating log files by hand, I couldn't design a test case that would violate it.